### PR TITLE
Added guide for Pheonix

### PR DESCRIPTION
**Pull request by sourabh1031**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> [I have taken reference from other resources [guide 1](http://phoenixframework.org/),[guide 2](https://medium.com/@elviovicosa/5-reasons-you-should-use-phoenix-instead-of-rails-in-your-next-project-504b4d83c48e),[guide 3](https://github.com/phoenixframework/phoenix)]

### CODE REVIEW

**Comment by sourabh1031** [comment]
@the-ethan-hunt Please review it

**Comment by RN0311** [comment]
Nice work, @sourabh1031 :smile: 
@the-ethan-hunt : Reviewed.

File: guides/Web_Frameworks/Pheonix.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Pheonix
```
Comment (the-ethan-hunt) [inline guides/Web_Frameworks/Pheonix.md]: Typos for Phoenix in a lot of places

File: guides/Web_Frameworks/Pheonix.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Installaion
+The link for the installation is [here](https://hexdocs.pm/phoenix/installation.html#content).Pheonix has been provided with rich set of guides to understand the workflow.The link to the [documentation](https://hexdocs.pm/phoenix/Phoenix.html) is here.If you fallen in love with it at the very first sight you can also contribute to it [here](https://github.com/phoenixframework/phoenix).
+
+# What about learning Phoenix?
+There are dozens of resources available on the web.The best is if one can scratch the surface with some good tutorials isn't? There are few resources available one is [head start guide](www.phoenixforrailsdevelopers.com) and this is also a [good book](https://pragprog.com/book/phoenix/programming-phoenix).
+
+# Why Choosing Pheonix ?
```
Comment (the-ethan-hunt) [inline guides/Web_Frameworks/Pheonix.md]: I don't want basic advantages. I want the lucid explanation of how Phoenix works. :smile:

**Comment by sourabh1031** [comment]
@the-ethan-hunt  shall i remove these advantages portion now?

**Comment by sourabh1031** [comment]
@RN0311 @the-ethan-hunt  please review it

**Pull request merged**